hazelcast/src/main/java/com/hazelcast/config/NetworkConfig.java

```diff
@@ -180,7 +180,7 @@ public boolean isReuseAddress() {
      * This property should not be set to true on the Windows platform: see
      * <ol>
      * <li>http://bugs.sun.com/bugdatabase/view_bug.do?bug_id=6421091</li>
-     * <li>http://www.hsc.fr/ressources/articles/win_net_srv/multiple_bindings.html</li>
+     * <li>http://www.hsc.fr/resources/articles/win_net_srv/multiple_bindings.html</li>
```

Comment:
Neither URL works

Reply:
Let's keep the old one, I believe it worked in the past. It's French site, in French it's with "ss". Finding its current version shouldn't be a part of this PR.